extensions/sql-migration/config.json

```diff
@@ -1,12 +1,12 @@
 {
 	"downloadUrl": "https://github.com/Microsoft/sqltoolsservice/releases/download/{#version#}/microsoft.sqltools.migration-{#fileName#}",
 	"useDefaultLinuxRuntime": true,
-	"version": "5.0.20241014.2",
+	"version": "5.0.20241024.1",
```

Comment:
why this downgrade?

Reply:
not a downgrade , its the latest version.